ios/RNSScreenStackHeaderConfig.mm

```diff
@@ -378,8 +378,12 @@ + (UIImage *)loadBackButtonImageInViewController:(UIViewController *)vc withConf
         // in DEV MODE we try to load from cache (we use private API for that as it is not exposed
         // publically in headers).
         RCTImageSource *imageSource = [RNSScreenStackHeaderConfig imageSourceFromImageView:imageView];
-        RCTImageLoader *imageLoader = [subview.bridge moduleForClass:[RCTImageLoader class]];
-
+        RCTImageLoader *imageLoader =
```

Comment:
it's a bit weird that the config extracts the imageLoader from the subview instead of receiving the `contextContainer` in the init. It's ok for now, if it works, but this approach is not super clean. I expect that the config are created before the view is created, no?

Reply:
I don't think there is a reason. Most likely, because the we had access to bridge already there and someone just used it. 

Let's init header config with bridge and use it on Paper. 

Same thing applies to holding the weak pointer to the module in subview shadow node. We should keep it in header config's state.

Reply:
I was not around back then 😅 

My (educated) guess would be that `loadBackButtonImageInViewController` was class method in order to match the pattern in header config - most of other methods are class-bounded as well. I seriously doubt that migrating here to instance method will give us any hiccup. We should be fine on this front.